### ISSUE TITLE
markdownit not working

### ISSUE THREAD

**Issue by asifm** [nuxt-community/modules#124]
This used to work fine. I'm not sure, but it's possible that it broke after I updated nuxt to rc6.

```
TypeError: Cannot read property 'loaders' of undefined

  - index.js:15 Builder.extendBuild.config
    [xxx]/[@nuxtjs]/markdownit/index.js:15:20

  - utils.js:79 Builder.<anonymous>
    [xxx]/[nuxt]/lib/common/utils.js:79:8

  - client.config.js:223 Builder.webpackClientConfig
    [xxx]/[nuxt]/lib/builder/webpack/client.config.js:223:31

  - builder.js:364 Builder._callee5$
    [xxx]/[nuxt]/lib/builder/builder.js:364:46

  - builder.js:171 Builder._callee$
    [xxx]/[nuxt]/lib/builder/builder.js:171:16

```
Versions in use:  
- nuxtjs / markdownit: 1.1.0,
- nuxtjs / markdownit-loader: 1.1.1,
- vue: 2.4.2,
- nuxt: 1.0.0-rc6

**Comment by wi-cuckoo** [comment]
Oh, Can you figure it out. I have the same problem.

**Comment by bayssmekanique** [comment]
Fixed by #125
